src/g_text.c

```diff
@@ -1251,8 +1251,10 @@ static void text_displace(t_gobj *z, t_glist *glist,
     {
         t_rtext *y = glist_findrtext(glist, x);
         rtext_displace(y, glist->gl_zoom * dx, glist->gl_zoom * dy);
-        text_drawborder(x, glist, rtext_gettag(y),
-            rtext_width(y), rtext_height(y), 0);
+        sys_vgui(".x%lx.c move %sR %d %d\n", glist_getcanvas(glist), rtext_gettag(y),
+                 dx * glist->gl_zoom, dy * glist->gl_zoom);
+        sys_vgui(".x%lx.c move %siolet %d %d\n", glist_getcanvas(glist), rtext_gettag(y),
+                 dx * glist->gl_zoom, dy * glist->gl_zoom);
```

Comment:
I think there might be a way to do this w/o creating a new iolet tag.. i'll think about this

Reply:
I though about adding a global tag as I did with many others, but it got complicated and I gave up thinking it couldn't be done sanely. Please have a go at it though to make sure :)